src/RA_Core.cpp

```diff
@@ -852,28 +862,45 @@ API void CCONV _RA_CheckForUpdate()
         }
         else
         {
+            // basic_ostringstream would seem overkill here
+
+            // stringstream is around 500 bytes (i/o variants are a bit smaller)
+            // and string is around 28 bytes
+
+            // The solution is string_view, it's only 8 bytes and guarantees
+            // compile time string building (immutable/constexpr/read-only)
+
+            // NB: It cannot be static or built with std::string because either a
+            //     data race (deadlock) or pointer dangling (could be leak,
+            //     access violoation, etc.) will occur
+            constexpr tstring_view view{
+                TEXT("Error in download from retroachievements.org...\nPlease check your connection"
+                " settings or RA forums!\n")
+            };
+            static_assert(view.find(RA_HOST_URL)); // just incase it changed
+
```

Comment:
The `constexpr ` and `static_assert` is all you need as proof that it's compile time, it's very small.